qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -64,67 +79,32 @@ class CliffordUtils:
         (2, 2, 3, 3, 3, 3, 4, 4),
         (2, 2, 3, 3, 4, 4),
     ]
+    GENERAL_CLIFF_LIST = ["id", "h", "sdg", "s", "x", "sx", "sxdg", "y", "z", "cx"]
+    TRANSPILED_CLIFF_LIST = ["sx", "rz", "cx"]
 
-    def clifford_1_qubit(self, num):
+    @classmethod
+    def clifford_1_qubit(cls, num):
```

Comment:
I was planning to deprecate `CliffordUtils` because it uses a class just as a namespace and doesn't make any difference from defining a set of functions in a module unless you need to define a subclass for a particular RB experiment.

Reply:
We could remove `CliffordUtils` as a class, and just keep the methods. I think it is nicer  and more expressive to have all these methods grouped in a class, to indicate the common functionality.

Reply:
Right, but still these are just a collection of functions. I wonder why these must be class methods, i.e.

```python
from qiskit_experiments.library.randomized_benchmarking import clifford_utils

clifford_utils.clifford_1_qubits(...)
```
v.s.
```python
from qiskit_experiments.library.randomized_benchmarking.clifford_utils import CliffordUtils

CliffordUtils.clifford_1_qubits(...)
```